packages/vm/src/environment.rs

```diff
@@ -452,6 +519,7 @@ mod tests {
 
     fn make_instance(
         gas_limit: u64,
+        contract_addr: Option<Addr>,
```

Comment:
I don't know much about it, so I have a question. why do you add contract_addr?

Reply:
it is for the test pass_callstack this https://github.com/line/cosmwasm/blob/b83dbbc2d4956e1bfb762817fef11a597d306187/packages/vm/src/environment.rs#L1008.

There must be envs with different addresses, but the existing mock instances all generate the same address. So, the address can be assigned arbitrarily.